pkg/utils/reconciler.go

```diff
@@ -32,11 +32,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/yaml"
 )
 
+var	logger = NewLogger(true) //log in JSON format
+var logName = "utils"
```

Comment:
This var "logName" doesn't looks like its referenced.

Reply:
It is referenced.    
if (logger.IsEnabled(LogTypeError)) {
			logger.Log(CallerName(), LogTypeError, fmt.Sprintf("Failed to convert into runtime.Object, Error: %s ", err), logName)
		}
It is copying from your original code:
var log = logf.Log.WithName("utils")